HTML-19/HTML-19.js

```diff
@@ -0,0 +1,23 @@
+
+let ispisPolje = document.getElementById("ispisPolje");
+let submitDugme = document.getElementById("submit");
+submitDugme.addEventListener("click", preslovi);
+
+function preslovi(){
+        let ispis = [];
+        let input = document.getElementById("input").value;
+        input.split('').forEach(function(c) {
+            if (c === c.toUpperCase()) {
+                ispis.push(c.toLowerCase());
+            }
+            else  {
+                ispis.push(c.toUpperCase());
+            }
+        });
+        ispisPolje.innerText = ispis;

```

Comment:
In this case you're going to print an array of the characters, it should be 
`ispisPolje.innerText = ispis.join('');`